### ISSUE TITLE
Transfer repositories & NPM packages from mongodb-js to electron-userland

### ISSUE THREAD

**Issue by malept** [electron-userland/welcome#7]
Following on conversation from https://github.com/mongodb-js/electron-installer-dmg/pull/25#issuecomment-386624960:

@MarshallOfSound is interested in moving `electron-installer-dmg` and `electron-squirrel-startup` over, as they're used in Electron Forge.

I'm kind of interested in `electron-installer-zip` since there's a link to it in the Electron Packager "related" section and it needs some updates (`zip-folder` has issues, as we've found out in Forge)

CC: @imlucas @zeke

**Comment by zeke** [comment]
Sounds like a great idea. 👍 

I wonder what we should do about npm ownership. 🤔 

One way to handle it:

1. create  `electron-userland-bot` account on npm
1. create @electron-userland-bot account on GitHub
1. `npm owner add electron-userland-bot`
1. set up [semantic-release](https://ghub.io/semantic-release)

This way releases would happen automatically, no matter who's landing pull requests.

**Comment by malept** [comment]
> I wonder what we should do about npm ownership.

There's an `electron-userland` team in NPM that we should add more of the `-userland` admins (and -userland packages) to. Unfortunately only the package creator can move it, it seems.

**Comment by zeke** [comment]
> There's an electron-userland team in NPM

Cool. Can you add me to it?

**Comment by malept** [comment]
@zeke added.

**Comment by imlucas** [comment]
This is great :) A few 🤔:

1. Guessing we move these packages over from [mongodb-js npm org](https://www.npmjs.com/org/mongodb-js) to electron-userland?

2. Repos: transfer to electron-userland gh org?

3. For the electron-* packages not listed above but are dependencies of those listed: should we also transfer these over so you all can refactor them out, deprecate, and move to archive?

For more context: these were all created to get https://compass.mongodb.com/ off the ground by going through atom build scripts and breaking them out into sep modules w strong testing. @durran is in Berlin and @rueckstiess is in Sydney so if I broke the build, they’d be stuck for 1 day or more until I could patch.

Longer term: https://www.github.com/mongodb-js/hadron-build is ~equivalent to forge. So, might make sense for us to move to forge @durran

The [hadron-* projects](https://github.com/mongodb-js?utf8=%E2%9C%93&q=hadron) might be a bit higher level for electron-userland but curious to discuss and I haven’t had ☕️ yet. Hadron provides the groundwork for building plugin based electron apps like atom, compass, and nylas. Would that be out of scope for electron-userland? IMO, hadron is incomplete today in terms of its aspirations. It would be sweet to get more community involvement here so it can be completed and enable others to build plugin based electron apps.

cc @aydrian @durran @rueckstiess @mrlynn

**Comment by zeke** [comment]
☝️ that all sounds good to me, @imlucas

**Comment by MarshallOfSound** [comment]
cc @imlucas Can we move forward with this plan?

**Comment by zeke** [comment]
Plan looks good to me. Let me know if you need my help for any of it.

**Comment by durran** [comment]
I’ll get on it this tomorrow.

**Comment by durran** [comment]
Just as a heads up here. I need temporary admin permissions in the electron-userland org (at least repo creation) to be able to transfer the repos over on Github. Once the transfer is complete please feel free to revoke my permissions.

**Comment by malept** [comment]
@durran invite sent.

**Comment by MarshallOfSound** [comment]
@durran Can you also add @malept and myself to the modules on NPM :+1:

**Comment by durran** [comment]
Those 3 I noted are now transferred. I'll do the npm access next.

**Comment by durran** [comment]
@MarshallOfSound Can you please provide yours and @malept's npm usernames and/or emails?

**Comment by durran** [comment]
If not here you can email me directly at durran@gmail.com or durran@mongodb.com

**Comment by MarshallOfSound** [comment]
Mine is the same as my github username.

https://www.npmjs.com/~marshallofsound

**Comment by MarshallOfSound** [comment]
Malept is the same, https://www.npmjs.com/~malept

**Comment by durran** [comment]
Both should have permission on npm now.

**Comment by durran** [comment]
And since the Github repos are now transferred please take away my privs. ;)

**Comment by malept** [comment]
@durran I just checked and none of the three packages are in my list in the NPM UI.

**Comment by malept** [comment]
@durran you have been removed as requested 😄

**Comment by durran** [comment]
I just checked as well and see the same. I'll try again.

**Comment by durran** [comment]
I've temporarily given both of you admin access to the mongodb-js org and that seemed to work fine. For some reason my attempts to give both of your ownership to those repos both via the command line and in the web interface never worked. Maybe you can move them now?

**Comment by MarshallOfSound** [comment]
@durran I have added those 3 modules successfully to the `electron-userland` org on NPM.  You should be able to remove them from the mongodb-js org now if you want 👍

**Comment by durran** [comment]
Great. Thank you so much!

**Comment by malept** [comment]
I was about to close this and I realized that we didn't move over `electron-squirrel-startup` yet. @MarshallOfSound do we still want to transfer this one?
